guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -6,45 +6,49 @@ You can execute a job that is based on a job template against one or more hosts.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-executing-a-remote-job_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the target hosts on which you want to execute a remote job.
-You can use the search field to filter the host list.
-. From the *Select Action* list, select *Schedule a Job*.
-. On the *Job invocation* page, define the main job settings:
-. Select the *Job category* and the *Job template* you want to use.
-. Optional: Select a stored search string in the *Bookmark* list to specify the target hosts.
-. Optional: Further limit the targeted hosts by entering a *Search query*.
-The *Resolves to* line displays the number of hosts affected by your query.
-Use the refresh button to recalculate the number after changing the query.
-The preview icon lists the targeted hosts.
-. The remaining settings depend on the selected job template.
-See {ManagingHostsDocURL}creating-a-job-template_managing-hosts[Creating a Job Template] for information on adding custom parameters to a template.
-. Optional: To configure advanced settings for the job, click *Display advanced fields*.
-Some of the advanced settings depend on the job template, the following settings are general:
-
-* *Effective user* defines the user for executing the job, by default it is the SSH user.
-
-* *Concurrency level* defines the maximum number of jobs executed at once, which can prevent overload of systems' resources in a case of executing the job on a large number of hosts.
-
-* *Timeout to kill* defines time interval in seconds after which the job should be killed, if it is not finished already.
-A task which could not be started during the defined interval, for example, if the previous task took too long to finish, is canceled.
-
-* *Type of query* defines when the search query is evaluated.
-This helps to keep the query up to date for scheduled tasks.
-
-* *Execution ordering* determines the order in which the job is executed on hosts: alphabetical or randomized.
+. In the {ProjectWebUI}, navigate to *Monitor* > *Jobs* and click *Run job*.
 +
-*Concurrency level* and *Timeout to kill* settings enable you to tailor job execution to fit your infrastructure hardware and needs.
+[NOTE]
+====
+{Project} {ProjectVersion} features a new job wizard.
+If you want to use job wizard from earlier {Project} versions, click *Use legacy form* in the top right corner of the job wizard.
+Note that this guide only applies to the new job wizard.
+====
+. Select the *Job category* and the *Job template* you want to use, then click *Next*.
+. Select hosts on which you want to run the job.
+You can filter hosts by host names, host collections, host groups, and a search query.
+If you do not select any hosts, the job will run on all hosts in the selected organization.
```

Comment:
_puts on pedantic hat_: It will run on all hosts that the user can see in the current context, taking permissions and current organization and location into consideration. Not sure if we need to make that distinction in the docs though

Reply:
I reworded the sentence, please see if the new form is better.